contracts/CrowdsaleExt.sol

```diff
@@ -441,23 +444,27 @@ contract CrowdsaleExt is Haltable {
 
   /**
    * Allow addresses to do early participation.
-   *
-   * TODO: Fix spelling error in the name
    */
-  function setEarlyParicipantWhitelist(address addr, bool status, uint minCap, uint maxCap) onlyOwner {
+  function setEarlyParticipantWhitelist(address addr, bool status, uint minCap, uint maxCap) onlyOwner {
     if (!isWhiteListed) throw;
-    earlyParticipantWhitelist[addr] = WhiteListData({status:status, minCap:minCap, maxCap:maxCap});
-    Whitelisted(addr, status);
+    assert(addr != address(0));
+    assert(minCap <= maxCap);
```

Comment:
@fvictorio sorry, Franco, I missed your previous comment. Yes, let's add a check, that cap values are positive. But, maybe we should check, that minCap is positive (not maxCap), because we've already checked, that maxCap >= minCap?

Reply:
Does it make sense to force `minCap` to be positive, though? As I see it, a positive `maxCap` makes sense (you wouldn't whitelist someone if they can't buy a single token), and a zero `minCap` makes sense too (maybe you want to let them buy as little as they want). What do you think?

Reply:
You are right.
How about 
```
assert(minCap >= 0);
assert(maxCap > 0);
```
?

Reply:
Since `minCap` is an `uint`, the `>= 0` check seems redundant, doesn't it?. I think just `assert(maxCap > 0)` and `assert(minCap <= maxCap)` should be enough, but I'm not completely sure.

Reply:
>Since minCap is an uint, the >= 0 check seems redundant, doesn't it?

Sure! nvm. Wait for an update with `assert(maxCap > 0)` and be able to merge.

Reply:
Done.